Pa3/Durchführung/Project/app/StateHandler.c

```diff
@@ -31,13 +31,190 @@
 /* PROTOTYPES *************************************************************************************/
 
 /* VARIABLES **************************************************************************************/
-static EventEnum gCurrentState;
+static EventEnum gCurrentEvent = NO_EVENT_HAS_HAPPEND; 
+static StateEnum gCurrentState = STATE_INIZALIZATION_MCU; //First state when the MCU is started
 static SoftTimer* gLapTimer;
+static bool gEntryDone = false;
 
 /* EXTERNAL FUNCTIONS *****************************************************************************/
 
 void StateHandler_process(void)
 {
+  switch(gCurrentState)
+  { 
+    case STATE_INIZALIZATION_MCU: 
+      InitializeMcu_InitializeAll(void); //entry
```

Comment:
in general, check the intendation level.
each level should be 4 spaces
please check your code against the CodingStyleGuide